Developer-guide/Backport-Guidelines.md

```diff
@@ -1,87 +1,26 @@
-Bugs often get fixed in master before release branches. When a bug is
-fixed in the master branch, it might be desirable or necessary in a
-stable branch. To put the fix in the stable branch we need to backport the
-fix to the stable branch.
+As a policy, any new change, bug fix, etc., are to be fixed in 'master'
+branch before release branches. When a bug is fixed in the master branch,
+it might be desirable or necessary in release branch.
 
-Anyone in the community can suggest a backport. If you are interested to
-suggest a backport, please check the [Backport
-Wishlist](./Backport-Wishlist.md).
+This page describes the policy GlusterFS has regarding the backports. As a user, or contributor, being aware of this policy would help you to understand how to request for backport from community.
 
-This page describes the steps needed to backport simple changes. Changes
-that do not apply cleanly will need some manual modifications and using
-`git cherry-pick` may not always be the easiest solution.
+## Policy
 
-## via Git command-line
-1.  Git clone the GlusterFS code
+* No feature from master would be backported to the release branch
+* Any CVE (aka, security alerts) on the existing releases would be backported, after getting fixed in master branch.
+* Only severe issues, crashes, data loss, or unavailability issues would be backported to the release.
+* For any other issues, upgrading to the higher version where bug is fixed is the recommended way.
+  - Gluster provides 'rolling' upgrade support, i.e., one can upgrade their version without stopping the application I/O, so we recommend migrating to higher version.
 
-                git clone ssh://username@review.gluster.org/glusterfs
+## Things to take care while backporting a patch.
```

Comment:
to pay attention to while backporting a patch.

Reply:
Done